test/functional/configuration/configuration.spec.js

```diff
@@ -407,6 +435,16 @@ test("restores default edge domain value when restore button is clicked", async
   await instances[0].edgeDomainField.expectValue(t, defaultEdgeDomain);
 });
 
+test("restores default download link qualifier when restore button is clicked", async t => {
+  await extensionViewController.init(t, defaultInitInfo);
+  await instances[0].downloadLinkQualifierField.typeText(t, ".");
+  await instances[0].downloadLinkQualifierRestoreButton.click(t);
+  await instances[0].downloadLinkQualifierField.expectValue(
+    t,
+    defaultDownloadLinkQualifier
+  );
+});
+
```

Comment:
You'll need to add to the "returns full valid settings" test. For the first Alloy instance, act like the user deselected the checkbox and for the other Alloy instance, act like the user provided a custom regex.

You'll also need to add a test to check that an error is shown for an invalid regular expression pattern (e.g., `[`), but only when click collection is enabled.